test/fixtures/test-runner/output/spec_reporter_cli.snapshot

```diff
@@ -180,14 +178,16 @@
       *
 
  callback called twice (*ms)
-  Error: callback invoked multiple times
```

Comment:
It would be nice to keep the `Error:` part.

Reply:
this it now the same as running without --test